### ISSUE TITLE
sparql-init-actor install `-g`: cannot find module '../../logger-pretty'

### ISSUE THREAD

**Issue by bjdmeest** [comunica/comunica#414]
#### Issue type:

- :bug: Bug

____
#### Description:

(a): the example command to start the SPARQL endpoint has a trailing `"`, but when removing this, it gives
(b): `Error: Cannot find module '../../logger-pretty'`

____
#### Environment:

```
$ comunica-sparql -v
internal/modules/cjs/loader.js:583
    throw err;
    ^

Error: Cannot find module '../../logger-pretty'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:581:15)
    at Function.Module._load (internal/modules/cjs/loader.js:507:25)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:22:18)
    at Object.<anonymous> (C:\Users\bjdmeest.Willaert\AppData\Roaming\npm\node_modules\@comunica\actor-init-sparql\lib\HttpServiceSparqlEndpoint.js:8:25)
    at Module._compile (internal/modules/cjs/loader.js:689:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:700:10)
    at Module.load (internal/modules/cjs/loader.js:599:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:538:12)
    at Function.Module._load (internal/modules/cjs/loader.js:530:3)
```

**Comment by rubensworks** [comment]
Hmm, there's probably some import somewhere that is referring the monorepo internally, instead of doing a proper package-based import.

@joachimvh Could you have a look at this?

**Comment by joachimvh** [comment]
Fixed in d199daa344679a1564b5c0274f75f641d9fb1249 so will be included in the next release.

**Comment by rubensworks** [comment]
Released as 1.6.4
